src/functional/program_processors/codegens/gtfn/itir_to_gtfn_ir.py

```diff
@@ -419,7 +425,7 @@ def visit_FunctionDefinition(
     def _is_scan(node: itir.Node):
         return isinstance(node, itir.FunCall) and node.fun == itir.SymRef(id="scan")
 
-    def _visit_output_argument(self, node: itir.SymRef | itir.FunCall):
+    def _visit_output_argument(self, node: itir.Expr | itir.FunCall):
```

Comment:
```suggestion
    def _visit_output_argument(self, node: itir.Expr):
```
since itir.FunCall is an expression already.